release notes/v0.6.0.md

```diff
@@ -1,49 +1,84 @@
 
-  Note that the `async` and `await` keywords are not yet supported (see [this k6 issue](https://github.com/grafana/k6/issues/779) for a workaround), so resolving the Promise using `then()` is required for scripts to continue to work correctly.
+  All our examples have been updated to work with async `page.goto`. For example, you can look at the [`browser_args.js` example](https://github.com/grafana/xk6-browser/blob/v0.6.0/examples/browser_args.js) for how to use it.
 
-  All our examples have been updated to work with async, or you can look at the [`browser_args.js` example](https://github.com/grafana/xk6-browser/blob/v0.6.0/examples/browser_args.js) for how to use it.
+  Note that the `async` and `await` keywords are not yet supported (see [this k6 issue](https://github.com/grafana/k6/issues/779) for a workaround), so resolving the `Promise` using `then()` is required for scripts to continue to work correctly.
 
+- Setting a browser launch property to `null` will be ignored. [#616](https://github.com/grafana/xk6-browser/pull/616))
 
+  For example:
+  ```js
+  const browser = chromium.launch({
+    headless: null,          // will be set to true
+    logCategoryFilter: null, // will be set to '.*'
+    timeout: nil,            // will be set to 30s
+  });
+  ```
+  
 ## Improvements
 
-- Added an example on how to use `xk6-browser` along side `k6`. ([#529](https://github.com/grafana/xk6-browser/pull/529))
+- Added an example of how to use `xk6-browser` alongside `k6`. ([#529](https://github.com/grafana/xk6-browser/pull/529))
 
 - Improved handling of browser process exit and WS connection closing. ([#535](https://github.com/grafana/xk6-browser/pull/535))
 
-- Improved how we handle (internal) events, such that they are dealt in the order we receive them in. This change has drastically improved the stability of the application. ([#555](https://github.com/grafana/xk6-browser/pull/555))
+  We're now waiting for the browser process to exit before ending the iteration. Previously, we would cancel the browser context prematurely, which caused the _process unexpectedly ended_ and _signal: killed_ errors.
+
+- Improved the internal event handling mechanism. ([#555](https://github.com/grafana/xk6-browser/pull/555))
+
+  Now we're handling events in the order we receive them. This change has drastically improved the extension's stability.
+
+- Improved emitting errors from the browser. [#598](https://github.com/grafana/xk6-browser/pull/598)
+
+  Now we're returning the error that can occur while launching a browser. Previously, we would log a _file already closed error_ if the browser process exits prematurely.
 
 - Added a Docker example. ([#556](https://github.com/grafana/xk6-browser/pull/556))
 
+  You can now use xk6-browser using Docker. For example:
+
+  ```bash
+  docker-compose run -T xk6-browser run - <examples/browser_on.js
+  ```
+
```

Comment:
It'd better to redirect users to the README 🤔

Reply:
I've added a link to the readme too.